cumulusci.yml

```diff
@@ -262,4 +286,26 @@ plans:
                     case_behavior_record_types:
                         name: EDA - Case Behavior Record Types
                 options:
-                    unmanaged: false
+                    unmanaged: False
+            4:
+                task: deploy_einstein_templates
+                ui_options:
+                    is_required: False
+                checks:
+                    - when: "tasks.check_einstein_licenses()"
+                      action: skip
+                      message: Skipped because Einstein Analytics permission sets are not available.
```

Comment:
It looks like you have the logic backward here? The check_einstein_licenses task returns true if 1 or more permset was found, and this check says to hide the step in that case.